tests/plugins/test_hide_files.py

```diff
@@ -52,93 +52,112 @@ def test_missing_config(self, tmpdir):
         # Verify Dockerfile contents have not changed
         assert df.content == df_content
 
-    @pytest.mark.parametrize('inherited_user', (None, "custom_user"))
-    @pytest.mark.parametrize('user_in_df', (None, "custom_docker_user"))
-    def test_hide_files(self, tmpdir, inherited_user, user_in_df):
-        if user_in_df:
-            df_content = dedent("""\
+    @pytest.mark.parametrize(('df_content', 'expected_df', 'inherited_user'), [
+        (
+            dedent("""\
```

Comment:
Since there's no indentation that needs to be kept consistent, would it improve readability if the dedents were removed from the parametrized arguments and moved to the function code itself?

Aside from that, this is a straightforward refactor.